api/src/middlewares/jwt.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response, NextFunction } from 'express';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const secret = process.env.ACCESS_TOKEN_SECRET as jwt.Secret;
+
+export const signAccessToken = (userId: string) => {
+    const payload = { 'sub': userId };
+    return jwt.sign(payload, secret);
```

Comment:
I would suggest to add expiresIn property here, to have better control over auth behavior